task/thermal/src/bsp/sidecar_ab.rs

```diff
@@ -22,13 +22,26 @@ use i2c_config::sensors;
 
 task_slot!(SEQUENCER, sequencer);
 
+////////////////////////////////////////////////////////////////////////////////
+// Constants!
+
+// Air temperature sensors, which aren't used in the control loop
 const NUM_TEMPERATURE_SENSORS: usize = sensors::NUM_TMP117_TEMPERATURE_SENSORS;
+
+// Temperature inputs (I2C devices), which are used in the control loop.
 pub const NUM_TEMPERATURE_INPUTS: usize =
     sensors::NUM_TMP451_TEMPERATURE_SENSORS;
+
+// External temperature inputs, which are provided to the task over IPC
+// In practice, these are our transceivers.
+pub const NUM_DYNAMIC_TEMPERATURE_INPUTS: usize = 32;
```

Comment:
Could this be `transceivers_api::NUM_PORTS`?

Reply:
Yeah, done in 1829494e21c133fc6dfcf5e26bc9b9c480ba94a8

(although this task would still need edits if it changed, because of the new `DYNAMIC_INPUTS` array below, which arrived after your review)